docs/source/notebook/plotting.rst

```diff
@@ -0,0 +1,36 @@
+.. _plotting:
```

Comment:
This entire folder is being removed by #8695, so this document needs to go somewhere else. Probably the `interactive` subfolder.